pkg/sriov/operatorconfig.go

```diff
@@ -130,13 +130,13 @@ func (builder *OperatorConfigBuilder) Exists() bool {
 	return err == nil || !k8serrors.IsNotFound(err)
 }
 
-// WithInjector configures enableInjector in the SriovOperatorConfig.
+// WithInjector configures replace in the SriovOperatorConfig.
 func (builder *OperatorConfigBuilder) WithInjector(enable bool) *OperatorConfigBuilder {
 	if valid, _ := builder.validate(); !valid {
 		return builder
 	}
 
-	glog.V(100).Infof("Configuring enableInjector %t to SriovOperatorConfig object %s",
+	glog.V(100).Infof("Configuring replace %t to SriovOperatorConfig object %s",
```

Comment:
```suggestion
	glog.V(100).Infof("Configuring replace %t in SriovOperatorConfig object %s",
```

Reply:
sure, thank you! I have no idea why it was changed at all